packages/api/lib/rulesHelpers.js

```diff
@@ -104,11 +104,49 @@ function lookupCollectionInEvent(eventObject) {
   });
 }
 
+/**
+ * Build payload from rule for lambda invocation
+ *
+ * @param {RuleRecord} rule           - API rule
+ * @param {Object} [cumulusMeta]      - Optional cumulus_meta object
+ * @param {string} [asyncOperationId] - Optional ID for asynchronous operation
+ *
+ * @returns {Object} lambda invocation payload
+ */
+async function buildPayload(rule, cumulusMeta) {
```

Comment:
This was confusing. At some point we made this `buildPayload` function in rulesHelpers but we ALSO had a `buildPayload` in the Rule model. It looks like we switched everything over to use the helper instead of the model but the helper didn't set the `asyncOperationId` (line 141) correctly. It's updated here and this is the only rule-related `buildPayload` now.